### PR TITLE
:recycle: Pushing banners down to the data layer

### PR DESCRIPTION
**Pull request by morganick**
# Description

This is just a quick refactor so we aren't depending on _magic_ indexes for whether we show a banner on a particular schedule day.

# Graphics

No visual changes.

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

File: app/services/api/webflow-api.ts [hunk 1]
```diff
@@ -122,6 +122,7 @@ export const useScheduleScreenData = () => {
     isRefetching,
     schedules: [
       {
+        bannerTx: "scheduleScreen.workshopBanner",
```
Comment (mazenchami) [inline app/services/api/webflow-api.ts:125]: great idea. do we need to update the types somewhere also?

Comment (morganick) [reply app/services/api/webflow-api.ts:125]: This is of type `Schedule`. I've adjusted that type above for the optional `bannerTx`.

Comment (mazenchami) [reply app/services/api/webflow-api.ts:125]: ahh missed it, sorry.

**Review by mazenchami — APPROVED** [review]
♻️ 🚀

**Pull request merged**